public/assets/styles/site.css

```diff
@@ -0,0 +1,195 @@
+/* Oh Well */
+
+:root {
+  --foreground: #FFFFFF;
+  --background: #212121;
+  --background-dark: #000000;
+  --dark-accent-0: #444444;
+  --dark-accent-1: #333333;
+  --dark-accent-2: #888888;
+  --dark-accent-3: #4E4E4E;
+  --background-dark-transparent: #0000004d;
+  --red: #FF0000;
+}
+
+
+/* Define a rule for our custom font */
+@font-face {
+  font-family: 'badhandwriting';
+  font-style: normal;
+  font-weight: normal;
+  src: local('badhandwriting'),
+    url('fonts/badhandwriting.woff') format('woff'),
+    url('fonst/badhandwriting.ttf') format('truetype');
+}
+
+/* Apply the custom font to all navigation and titles */
+* {
+  font-family: 'badhandwriting', cursive, sans-serif !important;
+}
+
+/* Grid Styling */
+.header { grid-area: header; }
+.menu { grid-area: menu; }
+.main { grid-area: main; }
+.sidebar { grid-area: sidebar; }
+.footer { grid-area: footer; }
+.container {
+  display:grid;
+  grid-template-areas: 
+    'menu header header header header header'
+    'menu main main main main main'
+    'menu footer footer footer footer footer';
+  gap: 10px;
+}
```

Comment:
Here we're styling out the areas; 

On line 39-42; you'll see we're creating a kind of structure to your pages that use this stylesheet. 

We're establishing that your site is 6 grid columns wide in the first line; then we just need to name what each column should have in it. 
I made the MENU take up ALL of the left, with the rest of the site taking up the remainder of the real-estate. 

If you wanted the header and/or footer to overlap the menu; it would look like
```css
.container {
  display:grid;
  grid-template-areas: 
    'header header header header header header'
    'menu main main main main main'
    'footer footer footer footer footer footer';
  gap: 10px;
}
```